pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -74,6 +76,15 @@ type RepoSyncReconciler struct {
 	controller *controller.Controller
 }
 
+const (
+	// the following annotations are added to ConfigMaps that are copied from references
+	// in spec.helm.valuesFileRefs to the config-management-system namespace, to keep track
+	// of where the ConfigMap came from
+	repoSyncNameAnnotationKey          = configsync.ConfigSyncPrefix + "repo-sync-name"
+	repoSyncNamespaceAnnotationKey     = configsync.ConfigSyncPrefix + "repo-sync-namespace"
+	originalConfigMapNameAnnotationKey = configsync.ConfigSyncPrefix + "original-configmap-name"
```

Comment:
Where is it used? Did I miss it?

Reply:
Maybe we could argue that this is for debugging / referencing purpose.

Reply:
The first two are used in during [this cleanup](https://github.com/GoogleContainerTools/kpt-config-sync/blob/6666f12ee97eced1db7fc147cc3b335c9cebea91/pkg/reconcilermanager/controllers/reposync_controller.go#L1191) here so that we can check if this ConfigMap was created for this RepoSync before deleting it. Since the name is a hash, we have to rely on these annotations for that. 

The last one isn't actually used anywhere, but yes I thought it would be useful for debugging/referencing purposes.

Reply:
Added a comment to the code to clarify this